MovieMVVM/MovieMVVM/Services/CoreDataService/CoreDataService.swift

```diff
@@ -0,0 +1,165 @@
+// CoreDataService.swift
+// Copyright © Ilentiy. All rights reserved.
+
+import CoreData
+
+/// Core Data сервис
```

Comment:
```suggestion
/// Сервис по сохранению данных в CoreData
```
я бы так назвал, коммент на вкус и цвет